### ISSUE TITLE
Support Verkle tree prototype

### ISSUE THREAD

**Issue by terencechain** [prysmaticlabs/prysm#11553]
We should support the Verkle tree effort such that proofs and witnesses of the pre-state are included in the execution payload and the blocks. The new fields should propagate across the network so other clients can use them in their Verkle tree-enabled EL. We'll have to make modifications to the `ExecutionPayload` to support the following fields:
- `VerkleProof`: variable length byte array
- `VerkleKeyVals`: key-value pairs array. Key is a 32 byte vector, and the value can be a 32 byte list

cc @gballet

**Comment by potuz** [comment]
Assigning this to me as I've already spoken with @gballet and will produce a branch that simply changes the Bellatrix types for the time being

**Comment by gballet** [comment]
Thanks for creating the issue, I had a talk with @potuz and I'm happy to support this effort.

To make sure we are on the same page: the first step would be a prototype that starts in PoW mode on the EL side, with verkle trees already enabled there, and then perform the merge. Is that correct/what you have in mind?

**Comment by potuz** [comment]
Correct, we can start with pow genesis with Verkle trees and then merge directly without changing the execution payload object

**Comment by rkapka** [comment]
Closing as Verkle trees are not longer on the Ethereum roadmap
